index.html

```diff
@@ -148,7 +149,41 @@ <h2>Multi-Post Stories</h2>
             <li><a href="#">Bootstrap</a></li>
             <li><a href="#">Ruby</a></li>
         </ul>
-        <input type="submit" value="See Project" class="project-button">
+        <button type="submit" class="project-button">See Project</button>
+    </section>
+
+    <section class="contact-form">
+        <div>
```

Comment:
**Optional:**

- [ ] Currently, you have a horizontal scrollbar on your page. Kindly fix it. As the previous reviewer didn't ask for this change, I kept it optional but implementing this will make your page more responsive. :+1: 

https://user-images.githubusercontent.com/70483835/165505399-0d210e47-ae21-4f7d-8a07-dc77cc9a5fb7.mp4